modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1093,40 +1565,52 @@
      * @deprecated Use {@link IgniteConfiguration#getRebalanceThrottle()} instead.
      */
     @Deprecated
+    @SystemProperty(value = "Overrides CacheConfiguration.getRebalanceThrottle " +
+        "configuration property for all caches in milliseconds. 0 by default, which means that override is disabled",
+        type = Long.class, defaults = "0")
     public static final String IGNITE_REBALANCE_THROTTLE_OVERRIDE = "IGNITE_REBALANCE_THROTTLE_OVERRIDE";
 
     /**
      * Enables start caches in parallel.
      *
      * Default is {@code true}.
      */
+    @SystemProperty(value = "Enables start caches in parallel",
+        defaults = "" + DFLT_ALLOW_START_CACHES_IN_PARALLEL)
     public static final String IGNITE_ALLOW_START_CACHES_IN_PARALLEL = "IGNITE_ALLOW_START_CACHES_IN_PARALLEL";
 
     /** For test purposes only. Force Mvcc mode. */
+    @SystemProperty("For test purposes only. Force Mvcc mode")
     public static final String IGNITE_FORCE_MVCC_MODE_IN_TESTS = "IGNITE_FORCE_MVCC_MODE_IN_TESTS";
 
     /**
      * Allows to log additional information about all restored partitions after binary and logical recovery phases.
      *
      * Default is {@code true}.
      */
+    @SystemProperty("Allows to log additional information about all restored partitions after " +
+        "binary and logical recovery phases")
     public static final String IGNITE_RECOVERY_VERBOSE_LOGGING = "IGNITE_RECOVERY_VERBOSE_LOGGING";
 
     /**
      * Disables cache interceptor triggering in case of conflicts.
      *
      * Default is {@code false}.
      */
+    @SystemProperty("Disables cache interceptor triggering in case of conflicts")
     public static final String IGNITE_DISABLE_TRIGGERING_CACHE_INTERCEPTOR_ON_CONFLICT = "IGNITE_DISABLE_TRIGGERING_CACHE_INTERCEPTOR_ON_CONFLICT";
 
     /**
      * Sets default {@link CacheConfiguration#setDiskPageCompression disk page compression}.
      */
+    @SystemProperty(value = "Disk page compression - CacheConfiguration#setDiskPageCompression",
+        type = DiskPageCompression.class)
     public static final String IGNITE_DEFAULT_DISK_PAGE_COMPRESSION = "IGNITE_DEFAULT_DISK_PAGE_COMPRESSION";
 
     /**
      * Sets default {@link DataStorageConfiguration#setPageSize storage page size}.
      */
+    @SystemProperty(value = "Storage page size - DataStorageConfiguration#setPageSize", type = Integer.class)
     public static final String IGNITE_DEFAULT_DATA_STORAGE_PAGE_SIZE = "IGNITE_DEFAULT_DATA_STORAGE_PAGE_SIZE";
```

Comment:
The `IGNITE_DEFAULT_DATA_STORAGE_PAGE_SIZE` has default value `0`. Should we add it here?

Reply:
0 is the default value for the int variable.
So, it seems to me that constant will be useless here.